tests/test_speed.py

```diff
@@ -59,7 +66,7 @@ async def test_readwrite_speed(self):
         This isn't really a unit test, but running it as a test
         avoids bitrot.
         """
-        ntowrite = 500
+        ntowrite = 1000
```

Comment:
picky comment could it be `n_to_write` instead? I read it "not write" the first time.. hehe

Reply:
I'll change it to num_to_write